### PR TITLE
Update release notes draft to version v1.21.0-alpha.1

### PR DESCRIPTION
**Pull request by wilsonehusin**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.21.0-alpha.1

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.21/release-notes-draft.json [hunk 1]
```diff
@@ -0,0 +1 @@
...s/pull/97967), [@rajibmitra](https://github.com/rajibmitra)) [SIG Cloud Provider]","author":"rajibmitra","author_url":"https://github.com/rajibmitra","pr_url":"https://github.com/kubernetes/kubernetes/pull/97967","pr_number":97967,"areas":["provider/gcp"],"kinds":["cleanup"],"sigs":["cloud-provider"]},"97980":{"commit":"e0b2787ee1e066287d2d19c3841f18f07669b47d","text":"Fix the regression with the slow pods termination. Before this fix pods may take an additional time to terminate - up to one minute. Reversing the change that ensured that CNI resources cleaned up when the pod is removed on API server.","markdown":"Fix the regression with the slow pods termination. Before this fix pods may take an additional time to terminate - up to one minute. Reversing the change that ensured that CNI resources cleaned up when the pod is removed on API server. ([#97980](https://github.com/kubernetes/kubernetes/pull/97980), [@SergeyKanzhelev](https://github.com/SergeyKanzhelev)) [SIG Node]","author":"SergeyKanzhelev","author_url":"https://github.com/SergeyKanzhelev","pr_url":"https://github.com/kubernetes/kubernetes/pull/97980","pr_number":97980,"areas":["kubelet"],"kinds":["bug"],"sigs":["node"]}}
```
Comment (puerco) [inline releases/release-1.21/release-notes-draft.json:1]: We need to prettify this before 1.21

File: releases/release-1.21/release-notes/maps/pr-92817-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 92817
+releasenote:
+  do_not_publish: true
```
Comment (puerco) [inline releases/release-1.21/release-notes/maps/pr-92817-map.yaml:3]: Yes! The note was removed from the draft successfully, but it is in the json file. Another one to fix before 1.21

**Review by puerco — APPROVED** [review]
Thanks @wilsonehusin , I found two non blocking issues but the draft has your changes in :+1:

**Review by savitharaghunathan — COMMENTED** [review]
Thanks for the PR, @wilsonehusin 

/lgtm
/approve

**Pull request merged**